magit-todos.el

```diff
@@ -207,6 +207,14 @@ items."
                  (integer :tag "Automatically, but cache items for N seconds")
                  (const :tag "Manually" nil)))
 
+(defcustom magit-todos-update-remote nil
+ "Automatically scan for to-dos in remote repositories (i.e.
+accessed via TRAMP).
+
+Unless this option is set, any `magit-todos-update` setting for
+automatic scanning is ignored for remote repositories."
+  :type 'boolean)
```

Comment:
The most important thing is that the first line be a complete sentence; this is one of the Elisp guidelines.

Other than that, I'd probably write it something like this:

```
"Automatically scan in remote repositories.
Remote repositories include ones accessed via TRAMP (i.e. any
path for which `file-remote-p' returns non-nil).  If nil, remote
repos are only scanned when a manual scan is initiated (see
command `magit-todos-update')."
```